src/Element.scroll.ts

```diff
@@ -1,22 +1,16 @@
-import { IAnimationOptions, IContext, IScrollToOptions, modifyPrototypes, now, step } from "./common.js";
-
-let $original: (x: number, y: number) => void;
-
-export const getOriginalFunc = () => {
-    if ($original === undefined) {
-        $original =
-            HTMLElement.prototype.scroll ||
-            HTMLElement.prototype.scrollTo ||
-            function (this: Element, x: number, y: number) {
-                this.scrollLeft = x;
-                this.scrollTop = y;
-            };
-    }
-    return $original;
-};
+import {
+    IAnimationOptions,
+    IContext,
+    IScrollToOptions,
+    modifyPrototypes,
+    now,
+    original,
+    step,
```

Comment:
I'd like to change `step`, `original`, `now` to more descriptive variable names. It's hard to understand from my persperctive what exaclly do this part of functions, without digging inside the `common.js` 😿

Reply:
The variable naming is the hardest part :(
I feel a headache when I want to name it. And these functions are only for internal use. So I kept it as it was.